### DISCUSSION TITLE
Warning when deleting members if referenced on content nodes.

### DISCUSSION THREAD

**Discussion by meyntony** [umbraco/Umbraco-CMS#13781] [Features and ideas]
When a member is deleted, if it is referenced on content using the member picker data type, it would be nice to warn the backoffice user about these references in the delete flow.
![image](https://user-images.githubusercontent.com/85172999/216563927-0008bc20-ff17-480c-881b-618451dacde2.png)

Something similar to what we have when we delete referenced content as shown below:
![image](https://user-images.githubusercontent.com/85172999/216566472-9ef0b8e9-f682-49fd-800a-ca0315a75d5c.png)

**Comment by bergmania** [comment]
Make sense.. Should "just" be to do like we already do for media and content, so a new relating type should be created. I'll mark it for Up for grabs for now
